### PR TITLE
No cp phpstan (phpstan.phar without ext)

### PR DESCRIPTION
**Pull request by mvorisek**
merge after https://github.com/phpstan/phpstan/pull/4390

1/2 PR for https://github.com/phpstan/phpstan/issues/4389

### CODE REVIEW

File: compiler/build/scoper.inc.php [hunk 1]
```diff
@@ -26,12 +26,6 @@
 	'finders' => [],
 	'files-whitelist' => $stubs,
 	'patchers' => [
-		function (string $filePath, string $prefix, string $content): string {
```
Comment (mvorisek) [inline compiler/build/scoper.inc.php:29]: this is not needed, git blame does not advised any need/issue in the past with it

if so, please let me know

Comment (ondrejmirtes) [reply compiler/build/scoper.inc.php:29]: /cc @lookyman Could you remember what was the reason to include this patcher in the original phpstan-compiler? :) Thanks.

**Comment by mvorisek** [comment]
is the CI error related with this PR?

**Comment by ondrejmirtes** [comment]
No.

File: phpstan [hunk 2]
```diff
@@ -0,0 +1,6 @@
+#!/usr/bin/env php
+<?php
+
+declare (strict_types=1);
+
+require_once __DIR__ . '/bin/phpstan';
```
[the remark this answers is not in the document]
Comment (mvorisek) [reply phpstan:6]: see files in https://github.com/phpstan/phpstan-src/tree/master/bin , no

**Comment by ondrejmirtes** [comment]
I cherry-picked the relevant commit, thank you.

**Pull request closed without merging**